Makefile

```diff
@@ -32,15 +23,34 @@ upload:
 		docker push $$r/$(PROJECT):latest ; \
 	done
 
+CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
+.PHONY: controller-gen
+controller-gen:
+	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0)
```

Comment:
I believe since go 1.17 you can simply `go run sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0 ...` from anywhere without adding the dependency to `go.mod`. I'd prefer using that instead of manually implementing "package management" in the makefile

Reply:
Yeah you're right.
This part was taken from the last operator-sdk generated Makefile.